Library/ViewModels/PledgeCTAContainerViewViewModelTests.swift

```diff
@@ -294,6 +294,10 @@ internal final class PledgeCTAContainerViewViewModelTests: TestCase {
       dataLakeTrackingClientProperties?["session_variants_optimizely"],
       "Event includes Optimizely properties"
     )
+    XCTAssertNotNil(
```

Comment:
Instead of asserting `NotNil` you should check for the exact value we are expecting.